distr_tracing/distr_tracing_install/distr-tracing-deploying-otel.adoc

```diff
@@ -9,3 +9,7 @@ toc::[]
 The {OTELName} Operator uses a custom resource definition (CRD) file that defines the architecture and configuration settings to be used when creating and deploying the {OTELName} resources. You can either install the default configuration or modify the file to better suit your business requirements. 
 
 include::modules/distr-tracing-config-otel-collector.adoc[leveloffset=+1]
+
+include::modules/distr-tracing-config-otel-send-data-to-monitoring-stack.adoc[leveloffset=+1]
```

Comment:
this module name could be shortened to match the actual title, e.g.
```
[id="distr-tracing-send-metrics-monitoring-stack_{context}"]
= Sending metrics to the monitoring stack
```

either way, it needs to match the title and context ID

Reply:
+1 done